
From: "Martin J. Bligh" <mbligh@aracnet.com>

arch-i386-kernel-smpc-gcc341-inlining-fix.patch seems to cause
the following warning:

include/asm-i386/mach-numaq/mach_ipi.h:4: warning: static declaration for `send_
IPI_mask_sequence' follows non-static. 

This should fix it (and make it match the other subarches):

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-i386/mach-numaq/mach_ipi.h |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN include/asm-i386/mach-numaq/mach_ipi.h~268-rc2-mm2-warning-on-numa-q include/asm-i386/mach-numaq/mach_ipi.h
--- 25/include/asm-i386/mach-numaq/mach_ipi.h~268-rc2-mm2-warning-on-numa-q	Mon Aug  2 17:02:49 2004
+++ 25-akpm/include/asm-i386/mach-numaq/mach_ipi.h	Mon Aug  2 17:02:49 2004
@@ -1,7 +1,7 @@
 #ifndef __ASM_MACH_IPI_H
 #define __ASM_MACH_IPI_H
 
-static inline void send_IPI_mask_sequence(cpumask_t, int vector);
+inline void send_IPI_mask_sequence(cpumask_t, int vector);
 
 static inline void send_IPI_mask(cpumask_t mask, int vector)
 {
_
